analyzer-service/app.py

```diff
@@ -66,51 +66,51 @@ def redeploy():
 @app.route("/analyzer/var/<ticker>")
 def valueAtRisk(ticker):
   '''value at risk'''
-  ticker, data = Calc.loadHistorical(ticker)
-  test = Calc.calculateVar(ticker, data)
+  ticker, data = loadHistorical(ticker)
```

Comment:
If we're calling the `loadHistorical` method for every route, shouldn't that just be part of the calculation function? Or is there a better way to do this?

Reply:
Yes, I am going to create a calculation function that that essentially does all calcs for each ticker in a given list but I wanted to talk to you about the chrontab stuff before doing that.